### PR TITLE
Bug/fix for lock bruteforce

### PR DESCRIPTION
**Pull request by BloodyPixy**
Please, take a look at the code.

### CODE REVIEW

File: TrustTests/Lock/View Controllers/LockEnterPasscodeViewControllerTests.swift [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Copyright DApps Platform Inc. All rights reserved.
+
+import XCTest
+
+class LockEnterPasscodeViewControllerTests: XCTestCase {
```
Comment (vikmeup) [inline TrustTests/Lock/View Controllers/LockEnterPasscodeViewControllerTests.swift]: can you add test case? Also there is a space in the file name, can you remove its?

**Comment by OlegGordiichuk** [comment]
@BloodyPixy hmm nice work. Do you select same approach that we have discussed during Skype call ? (About passcode and touch flow)

**Comment by BloodyPixy** [comment]
@OlegGordiichuk yes, as I proposed, that's how it will work.

File: Trust/Lock/ViewControllers/LockPasscodeViewController.swift [hunk 2]
```diff
@@ -6,11 +6,12 @@ class LockPasscodeViewController: UIViewController {
     var willFinishWithResult: ((_ success: Bool) -> Void)?
     let model: LockViewModel
     var lockView: LockView!
-    let lock = Lock()
+    let lock: Lock!
```
Comment (vikmeup) [inline Trust/Lock/ViewControllers/LockPasscodeViewController.swift]: can we avoid force unwrap?

**Pull request merged**